### PR TITLE
feat: render layout preview in iframe too

### PR DESCRIPTION
**Pull request by karrui**
Add PreviewIframe component to render Preview inside iframe, injecting built Tailwind CSS instead of runtime updates. Update preview-tw.css styles to enhance grid, margin, padding, border, background, text, and transition utilities. Add new utility classes for layout and responsiveness. Deprecate and replace some classes with updated names and variables for consistency with site theme.

### CODE REVIEW

**Comment by karrui** [comment]
* **#413** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/413?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @karrui and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/studio/public/assets/css/preview-tw.css [hunk 1]
```diff
@@ -1097,10 +1097,18 @@ video {
   z-index: 50;
 }
 
+.col-span-12 {
+  grid-column: span 12 / span 12;
+}
```
Comment (seaerchin) [inline apps/studio/public/assets/css/preview-tw.css:1102]: i think can use css `for loop` to ease this but leave it up to you!

**Review by seaerchin — APPROVED** [review]

File: apps/studio/public/assets/css/preview-tw.css (hunk 1 above)
Comment (karrui) [reply apps/studio/public/assets/css/preview-tw.css:1102]: this is autogenerated lol

**Comment by karrui** [comment]
DID MY FORMATTER GIVE UP

**Pull request merged**